tests/unit/App.spec.ts

```diff
@@ -1344,7 +1365,7 @@ describe('App as a PAID (pending) Incorporation Application', () => {
     expect(vm.$store.getters.getEntityName).toBe('My Name Request')
   })
 
-  it('fetches IA filing properly', () => {
+  it.only('fetches IA filing properly', () => {
```

Comment:
Don't commit this :)

Reply:
Oh no!

Reply:
I was wondering why the code cov was unhappy..

Reply:
Fixed in next commit